examples/swarmion-with-next/.github/workflows/merge-main.yml

```diff
@@ -26,43 +31,33 @@ jobs:
           fetch-depth: 0
       - uses: nrwl/nx-set-shas@v3
       - uses: pnpm/action-setup@v2.2.3
-      - name: Use Node.js ${{ env.NODE_VERSION }}
+      - name: Setup Node.js
+        id: setup-node
         uses: actions/setup-node@v3
         with:
           node-version-file: '.nvmrc'
           cache: 'pnpm'
       - name: Install dependencies
-        run: pnpm install
+        run: pnpm install --frozen-lockfile
       - name: 💾 Cache Nx cache
         id: package-cache
         uses: actions/cache@v3
         with:
           path: |
             nx-cache
           # Cache will be updated at every run: https://github.com/actions/cache/blob/main/workarounds.md#update-a-cache
-          key: ${{ runner.os }}-nx-cache-${{ env.NODE_VERSION }}-${{github.event.number}}-${{ github.run_id }}
+          key: ${{ runner.os }}-nx-cache-${{ steps.setup-node.outputs.node-version }}-${{ github.run_id }}
           restore-keys: |
-            ${{ runner.os }}-nx-cache-${{ env.NODE_VERSION }}-${{github.event.number}}-
-            ${{ runner.os }}-nx-cache-${{ env.NODE_VERSION }}-
+            ${{ runner.os }}-nx-cache-${{ steps.setup-node.outputs.node-version }}-
             ${{ runner.os }}-nx-cache-
       - name: '🏗 Package'
         run: pnpm nx affected --target=package --parallel=2
-      - name: '🧪 Test linter'
-        run: pnpm nx affected --target=test-linter --parallel=2
-      - name: '🧪 Test type'
-        run: pnpm nx affected --target=test-type --parallel=2
-      - name: '🧪 Test unit'
-        run: pnpm nx affected --target=test-unit --parallel=2
-      - name: '🧪 Test circular'
-        run: pnpm nx affected --target=test-circular --parallel=2
-      - name: 'Get serverless stack region'
-        id: get-stack-region
-        run: |
-          echo "AWS_REGION=$(pnpm --silent -C services/core run sls-get-region)" >> $GITHUB_OUTPUT
+      - name: '🧪 Run tests'
+        run: pnpm nx affected --targets=test-linter,test-type,test-unit,test-circular --parallel=2
       - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@master
+        uses: aws-actions/configure-aws-credentials@v1-node16
         with:
-          aws-region: ${{ steps.get-stack-region.outputs.AWS_REGION }}
-          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }} # Create the role in AWS and add it to the secrets
+          aws-region: ${{ env.AWS_REGION }}
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN_PRODUCTION }}
```

Comment:
We should change the docs to document that these should be defined/changed

Reply:
Done